res/css/views/auth/_AuthBody.scss

```diff
@@ -94,6 +94,22 @@ limitations under the License.
             background-color: $authpage-focus-bg-color;
         }
     }
+
+    .mx_recaptchaContainer {
```

Comment:
Components should have their own style files, since these classes are used in `CaptchaForm` component they should be in `_CaptchaForm.scss`. This helps keep our components independent and maintainable.

Please move these into the correct file (and don't forget to run `yarn rethemendex` when adding new scss files)

Reply:
Addressed: https://github.com/matrix-org/matrix-react-sdk/pull/8208/commits/42874a97bed6c482589a80f66f4c395ea8b8ce9f